internal/shim-sgx/src/handler.rs

```diff
@@ -564,4 +567,17 @@ impl<'a> Handler<'a> {
 
         Ok(result)
     }
+
+    // Stub for get_attestation() pseudo syscall
+    // See: https://github.com/enarx/enarx-keepldr/issues/31
+    pub fn get_att(&mut self) -> sallyport::Result {
+        self.trace("get_att", 0);
+
+        let rep = sallyport::Reply {
+            ret: [0.into(), SGX_TECH.into()],
+            err: Default::default(),
+        };
```

Comment:
You want this instead:

```
let rep: sallyport::Reply = Ok([0.into(), SGX_TECH.into()]).into();
```

Reply:
Thanks! I was looking for this and couldn't find the right way.